plonky2/Cargo.toml

```diff
@@ -31,7 +31,7 @@ plonky2_util = { path = "../util", default-features = false }
 rand = { version = "0.8.4", default-features = false }
 rand_chacha = { version = "0.3.1", optional = true, default-features = false }
 serde = { version = "1.0", default-features = false, features = ["derive", "rc"] }
-serde_json = "1.0"
+serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
```

Comment:
This actually should be moved to `[dev-dependencies]`, it's not being used by the library (hence why it compiles fine against wasm targets), only by the `fibonacci_serialization` example which does require `std` anyway.